src/pocketmine/event/server/RemoteServerCommandEvent.php

```diff
@@ -27,6 +27,8 @@
 
 /**
  * This event is called when a command is received over RCON.
+ *
+ * @deprecated Use CommandEvent instead.
```

Comment:
will this trigger the appropriate deprecation warnings when a plugin registers a handler for this?

Reply:
Not tested, but I think so